browser/src/control/Control.NotebookbarCalc.js

```diff
@@ -994,6 +994,11 @@ L.Control.NotebookbarCalc = L.Control.NotebookbarWriter.extend({
 				'text': _UNO('.uno:InsertObjectChart'),
 				'command': '.uno:InsertObjectChart'
 			},
+			{
+				'type': 'bigtoolitem',
+				'text': _UNO('.uno:InsertSparkline'),
```

Comment:
I think it would be better to avoid having the text from the uno command in the Insert tab because otherwise it would be "Insert Sparkline".

So probably best to replace this with ` _('Sparkline'),` so in does not have the "Insert" and it stays consistent with the other items on that specific tab

Reply:
I think this is ok right @eszkadev ?